crates/ruma-common/Cargo.toml

```diff
@@ -52,6 +49,9 @@ unstable-msc3553 = ["unstable-msc3552"]
 unstable-msc3554 = ["unstable-msc1767"]
 unstable-msc3786 = []
 unstable-msc3827 = []
+unstable-pdu = []
```

Comment:
accidental sort?

Reply:
No, intentional ^^

Reply:
what is `pdu` though and where do we document it?

Reply:
It's for homeserver stuff that isn't used by Conduit and I'm not quite sure about its design.